code-style/action.yml

```
@@ -49,3 +63,36 @@ runs:
     - name: "Run pre-commit"
       shell: bash
       run: pre-commit run --all-files --show-diff-on-failure
+
+    # The followings steps are applying docker-style action to allowed working dirs if the exist.
+    # Changes performed here should coherent with the 'allowed_working_dirs' in the docker-style action.
+    - name: "Check if docker directory exists"
+      shell: bash
+      id: checkdocker
+      run: |
+        if [ -d "${{ github.workspace }}/docker" ]; then
+          echo "exists=true" >> $GITHUB_OUTPUT
```

I would rename this var to:

```suggestion
          echo "HAS_DOCKER_DIR=true" >> $GITHUB_OUTPUT
```